instantsearch-android/src/main/java/com/algolia/instantsearch/helper/filter/list/FilterListConnector.kt

```diff
@@ -25,6 +30,13 @@ public abstract class FilterListConnector<T : Filter> : ConnectionImpl() {
         connectionFilterState.disconnect()
     }
 
+    /**
+     * [Documentation](https://www.algolia.com/doc/api-reference/widgets/filter-list-facet/android/)
```

Comment:
Cool for leaving a link to the public documentation :) 

Small remark is that I see 14 of them, but there are more than that in the [widget showcase page](https://www.algolia.com/doc/guides/building-search-ui/widgets/showcase/android/). Just Double check that you didn't miss a few widgets